ovn-tester/tests/service_route.py

```diff
@@ -0,0 +1,82 @@
+from collections import namedtuple
+from ovn_context import Context
+from ovn_workload import Namespace
+from ovn_ext_cmd import ExtCmd
+
+import netaddr
+import ovn_load_balancer as lb
+
+ServiceRouteCfg = namedtuple('ServiceRouteCfg',
+                             ['n_lb_cluster',
+                              'n_lb_node'])
+
+
+DEFAULT_VIP_SUBNET = netaddr.IPNetwork('90.0.0.0/16')
+DEFAULT_VIP_SUBNET6 = netaddr.IPNetwork('9::/32')
+DEFAULT_VIP_PORT = 80
+DEFAULT_BACKEND_PORT = 8080
+
+
+class ServiceRoute(ExtCmd):
+    def __init__(self, config, central_node, worker_nodes, global_cfg):
+        super(ServiceRoute, self).__init__(
+                config, central_node, worker_nodes)
+        test_config = config.get('service_route', dict())
+        self.config = ServiceRouteCfg(
+            n_lb_cluster=test_config.get('n_lb_cluster', 16),
+            n_lb_node=test_config.get('n_lb_node', 4)
+        )
+
+    def provide_vip(self, ip, vip_subnet, index):
```

Comment:
This seems very similar to the code in `DensityHeavy.create_lb()`. Could you please factor out the code into a static method, for example:
```
OvnLoadBalancer.provide_vip(ip, vip_subnet, index)
```

Reply:
@dceara not sure if it worth to do so with the current codebase, what do you think?

Reply:
@LorenzoBianconi Ok, we can refactor later I guess.